docs/general/performance.md

```diff
@@ -69,6 +69,42 @@ There are many approaches to data decimation and selection of an algorithm will
 
 Line charts are able to do [automatic data decimation during draw](#automatic-data-decimation-during-draw), when certain conditions are met. You should still consider decimating data yourself before passing it in for maximum performance since the automatic decimation occurs late in the chart life cycle.
 
+## Render Chart.js in a web worker (Chrome only)
+
+Chrome recently added the ability to [transfer rendering control of a canvas](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/transferControlToOffscreen) to a web worker. Web workers can use the [OffscreenCanvas API](https://developer.mozilla.org/en-US/docs/Web/API/OffscreenCanvas) to render from a web worker onto canvases in the DOM. Chart.js is a canvas-based library and supports rendering in a web worker - just pass an OffscreenCanvas into the Chart constructor instead of a Canvas element. Note that as of today, this API is only supported in Chrome.
+
+By moving all Chart.js calculations onto a separate thread, the main thread can be freed up for other uses. Some tips and tricks when using Chart.js in a web worker:
+* Transferring data between threads can be expensive, so ensure that your config and data objects are as small as possible. Try generating them on the worker side if you can (workers can make HTTP requests!) or passing them to your worker as ArrayBuffers, which can be transferred quickly from one thread to another.
+* You can't transfer functions between threads, so if your config includes functions make sure to strip them out before passing them across worker boundaries.
```

Comment:
Maybe it's still possible to use functions with web workers though if done properly? Perhaps we could update this line to something like:

You can't transfer functions between threads. This means if you can't pass a config with functions across worker boundaries. If you're passing the config from the main thread you'll need to strip out any functions. Alternatively, you can construct or fetch the config in the web worker.

Reply:
>Maybe it's still possible to use functions with web workers though if done properly?

Should be obvious.

To me the @davidswinegar's proposed text is clearer.

Reply:
I'll clarify this!